imports/react-ui/insights/containers/Insights.js

```diff
@@ -0,0 +1,26 @@
+import { Meteor } from 'meteor/meteor';
+import { Mongo } from 'meteor/mongo';
+import { compose } from 'react-komposer';
+import { getTrackerLoader, composerOptions } from '/imports/react-ui/utils';
+import { Brands } from '/imports/api/brands/brands';
+
+import { Insights } from '../components';
+const Integration = new Mongo.Collection('integration');
```

Comment:
Why is this collection created here? Collections must be declared in `api`. See the other modules such as `customers` and `conversations`.